cli/internal/run/run.go

```diff
@@ -727,6 +736,15 @@ func (r *run) initCache(ctx gocontext.Context, rs *runSpec, analyticsClient anal
 	apiClient := r.base.APIClient
 	// Theoretically this is overkill, but bias towards not spamming the console
 	once := &sync.Once{}
+	// rs.Opts.runOpts
+
+	useHTTPCache := !rs.Opts.cacheOpts.SkipRemote
+	if useHTTPCache {
+		r.logInfo("• Remote caching enabled")
+	} else {
+		r.logInfo("• Remote caching disabled")
+	}
```

Comment:
This used to be internal to `cache.New()`, but I felt like it was too complicated to pass `r` or create _another_ logger struct to pass it into the cache module.